gspread/worksheet.py

```diff
@@ -2209,7 +2213,7 @@ def _finder(
             str_query = query
 
             def match(x: Cell) -> bool:
-                if case_sensitive:
+                if case_sensitive or x.value is None:
```

Comment:
what is this?

Reply:
This for typing.

Just below we have the string compare method: `x.value.casefold()` and mypy tells me we can't do that if value is ` None` of course. It would result in an exception like `TypeError: 'NoneType' object is not callabl`.

And this will prevent us from it